arangod/Cluster/ClusterFeature.h

```diff
@@ -183,10 +195,10 @@ class ClusterFeature : public application_features::ApplicationFeature {
   ServerState::RoleEnum _requestedRole = ServerState::RoleEnum::ROLE_UNDEFINED;
   metrics::Histogram<metrics::LogScale<uint64_t>>& _agency_comm_request_time_ms;
   std::unique_ptr<network::ConnectionPool> _asyncAgencyCommPool;
-  std::optional<std::reference_wrapper<metrics::Counter>> _followersDroppedCounter;
-  std::optional<std::reference_wrapper<metrics::Counter>> _followersRefusedCounter;
-  std::optional<std::reference_wrapper<metrics::Counter>> _followersWrongChecksumCounter;
-  std::optional<std::reference_wrapper<metrics::Counter>> _followersTotalRebuildCounter;
+  metrics::Counter* _followersDroppedCounter = nullptr;
```

Comment:
Maybe we need an abstraction like `RawPtr<T>`? 
A class that encapsulates access to a pointer and adds an assertion that it isn't `nullptr` and adds default ctor.

Reply:
Sounds good!